### DISCUSSION TITLE
To download list of certificate to be expired ?

### DISCUSSION THREAD

**Discussion by rajeev-0** [Keyfactor/ejbca-ce#443] [Q&A]
Hi,

I would like to know if there is a way to download the list of certificate which are going to expire after certain date?
I could filter the certificates in  `ejbca/ra/search_certs.xhtml` by `Expires after`, But don't have option to download the list.

**Comment by primetomas** [comment]
Look at the search certificate REST API. 
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/ejbca-rest-interface#EJBCARESTInterface-SearchforCertificatesVersion2
